src/YAMLValidator/Utility.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// using YamlDotNet.Core;
```

Comment:
In general, we should remove commented out code
```suggestion
```

Some exceptions for work in progress, or awaiting some other change, may be fine, but should typically have an explanatory comment on why they're commented out.

Reply:
just a leftover from something I tried so I ended up deleting this . I did add a comment to this based on what you suggested https://github.com/microsoft/PowerApps-Tooling/blob/afaf6d739d9152483f6040a1a9ee84d552ac0a9b/src/YamlValidator/Validator.cs#L38 makes, sense this was